stretch_description/urdf/stretch_description.xacro

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="stretch_description">
 
-  <xacro:include filename="stretch_gripper.xacro" />
+  <xacro:include filename="stretch_dex_wrist.xacro" />

```

Comment:
@hello-chintan Could we have the Stretch Description configured for the Standard gripper itself instead of DexWrist by default (i.e. before running [configure_wrist.py](https://github.com/hello-robot/stretch_ros2/blob/feature/dexwrist/hello_helpers/src/hello_helpers/configure_wrist.py)) to match our bringup qc process.

Reply:
Sure! I'll update that.